pkg/utils/secrets/secrets.go

```diff
@@ -50,7 +52,7 @@ func (s *Secrets) Deploy(
 
 	// Generate cluster secrets
 	secretConfigs := s.SecretConfigsFunc(cas, namespace)
-	clusterSecrets, err := GenerateClusterSecrets(gcs, existingSecrets, secretConfigs, namespace)
+	clusterSecrets, err := GenerateClusterSecrets(context.TODO(), gcs, existingSecrets, secretConfigs, namespace)
```

Comment:
Can you let the `context` be passed to `func (s *Secrets) Deploy` (line 35) and use it here?

Reply:
done